src/infrastructure/gateway/generalPartner/GeneralPartnerGateway.ts

```diff
@@ -69,6 +57,51 @@ class GeneralPartnerGateway implements IGeneralPartnerGateway {
 
     return response?.resource ?? {};
   }
+
+  async sendPageData(
+    opt: { access_token: string; refresh_token: string },
+    transactionId: string,
+    generalPartnerId: string,
+    data: Record<string, any>
+  ): Promise<void> {
+    this.validAndFormatDateOfBirth(data);
+
+    // if there was former_names and the selection changed to false (Person)
+    if (data?.former_names && data?.previousNames === "false") {
+      data.former_names = "";
```

Comment:
So maybe we should send as a null/undefined value here rather than an empty string (to avoid the API validation issue)?